dev/bots/test.dart

```diff
@@ -730,7 +730,7 @@ Future<void> _runFrameworkTests() async {
     final List<String> tests = Directory(path.join(flutterRoot, 'packages', 'flutter', 'test'))
       .listSync(followLinks: false)
       .whereType<Directory>()
-      .where((Directory dir) => dir.path.endsWith('widgets') == false)
+      .where((Directory dir) => dir.path.endsWith('widgets') == false && dir.path.endsWith('html') == false)
```

Comment:
Filtering it out here like this means one can no longer just `cd packages/flutter` and run `flutter test`. Can we use the `@TestOn` annotation instead? See: https://pub.dev/documentation/test_api/latest/test_api.scaffolding/TestOn-class.html

If this is the only way to filter out html-specific tests, consider leaving a comment explaining why this is done here.

BTW, is there risk that this will inadvertently skip tests making it look like tests succeeded, while in fact we didn't run them?